service/rpc2/guess_substitute_path_test.go

```diff
@@ -0,0 +1,17 @@
+package rpc2
+
+import (
+	"runtime"
+	"testing"
+)
+
+func TestMakeGuessSusbtitutePathIn(t *testing.T) {
+	if runtime.GOARCH == "ppc64le" {
+		t.Setenv("GOFLAGS", "-tags=exp.linuxppc64le")
+	}
+	gsp, err := MakeGuessSusbtitutePathIn()
+	if err != nil {
+		t.Fatal(err)
+	}
+	t.Logf("%#v", gsp)
```

Comment:
Is it worth asserting about the actual response other than it didn't error out?

Reply:
Actually, this function is really being tested by TestGuessSubstitutePath in integration2_test, I think I had forgotten to delete it.